stdlib/universe/difference_test.go

```diff
@@ -23,17 +25,6 @@ func TestDifferenceOperation_Marshaling(t *testing.T) {
 	querytest.OperationMarshalingTestHelper(t, data, op)
 }
 
-func TestDifference_PassThrough(t *testing.T) {
```

Comment:
There wasn't an equivalent test helper for this for the updated `New*`signature (like there is for the other tests). Should I try to add one or is it not relevant anymore?

Reply:
Yea this test isn't relevant anymore